python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -802,8 +804,10 @@ def _doDiffIm(processed_repo, dataId, templateType, template, diffim_repo):
     log.info('Running ImageDifference...')
     if not os.path.isdir(diffim_repo):
         os.mkdir(diffim_repo)
-    diffim_result = ImageDifferenceTask.parseAndRun(args=args, config=config, doReturnResults=True)
-    diffim_metadata = diffim_result.resultList[0].metadata
+    ImageDifferenceTask.parseAndRun(args=args, config=config, doReturnResults=True)
```

Comment:
Since you're no longer grabbing the results of this call to `parseAndRun`, you can presumably drop the `doReturnResults`.